mws/apis/finances.py

```diff
@@ -24,26 +23,25 @@ class Finances(MWS):
     def list_financial_event_groups(
         self, created_after=None, created_before=None, max_results=None, next_token=None
     ):
-        """
-        Returns financial event groups for a given date range.
+        """Returns financial event groups for a given date range.
         If `created_before` is ommitted, defaults to now minus 2 minutes.
 
         Pass `next_token` to call "ListFinancialEventGroupsByNextToken" instead.
 
         Docs:
         http://docs.developer.amazonservices.com/en_US/finances/Finances_ListFinancialEventGroups.html
         """
-        data = {
-            "Action": "ListFinancialEventGroups",
-            "FinancialEventGroupStartedAfter": created_after,
-            "FinancialEventGroupStartedBefore": created_before,
-            "MaxResultsPerPage": max_results,
-        }
-        return self.make_request(data)
+        return self.make_request(
```

Comment:
We seem to have changed from building a data object, and then passing this to the `make_request` function, to sending the `action` and then the dictionary of data (all good, no issues here) but then in inbound shipment `get_inbound_guidance_for_sku` we go back to the old way.

I can see why, as we have to adjust some of the items in data before hand, but it could get confusing for future debugging when the 'why this way here' question etc etc.

Small but worth mentioning.